apps/src/craft/code-connection/cc-client.js

```diff
@@ -18,19 +35,23 @@ export default class cc_client {
      *
      */
     connectionStatusUpdate(asyncCallback, timeout = 200) {
-        $.ajax({
-            type: "GET",
-            url: `${baseUrl}${this.port}/connected`,
-            timeout: timeout,
-            success: function (data) {
-                // cc responded
-                asyncCallback(data);
-            },
-            error: function (jqxhr, textStatus, error) {
-                // TODO: handle net::ERR_CONNECTION_REFUSED error gracefully
-                asyncCallback(false);
-            }
-        });
+        if (window.ipcRenderer !== undefined) {
+            asyncCallback(window.ipcRenderer !== undefined);
```

Comment:
note that this will now always be `true`

Reply:
Roger that. I think that's the state it needs to be in if we're using the ipcRenderer (I am by no means a seasoned expert with Electron apps, or the cc-client async function call). I did this work about a month ago, so the memory is a bit hazy, but I believe it wasn't working correctly without that setting. If in testing something comes up and there is bad behavior somewhere we can absolutely change stuff.

Just let me know 👍